ci-operator/step-registry/openshift/e2e/test/clusterinfra-qe/regression/openshift-e2e-test-clusterinfra-qe-regression-chain.yaml

```diff
@@ -0,0 +1,32 @@
+chain:
+  as: openshift-e2e-test-clusterinfra-qe-regression
+  env:
+  - name: E2E_RUN_TAGS
+    default: ""
+    documentation: |
+      It is used to filter the cucushift cases.
+  - name: TEST_FILTERS
+    default: NonHyperShiftHOST
+    documentation: |
+      It is used to filter the openshift-test-private cases.
+  - name: TEST_FILTERS_CLUSTERINFRASTRUCTURE
+    default: "!qe-only"
+    documentation: |
+      It is used to filter the cluster-api-actuator-pkg cases.
+  - name: TEST_SCENARIOS
+    default: Cluster_Infrastructure
+    documentation: |-
+      It is used to select the openshift-test-private cases. It's a RE pattern and is passed to the 'grep -E'
```

Comment:
can we add more filters as example in documentation?

Reply:
good idea, added, thanks!